lib/workers/global/initialize.ts

```diff
@@ -76,7 +76,9 @@ export async function globalInitialize(
   limitCommitsPerRun(config);
   setEmojiConfig(config);
   setGlobalHostRules(config);
-  await initMergeConfidence();
+  await initMergeConfidence(
+    process.env.RENOVATE_X_MERGE_CONFIDENCE_SUPPORTED_DATASOURCES,
```

Comment:
Why does this need to be passed as a parameter? Couldn't it read the env itself?

Reply:
It makes it much easier to test, but sure we can just use env instead.